src/main/java/uk/gov/cshr/repository/InviteRepository.java

```diff
@@ -5,13 +5,17 @@
 import uk.gov.cshr.domain.Invite;
 import uk.gov.cshr.domain.InviteStatus;
 
+import java.util.Optional;
+
 @Repository
 public interface InviteRepository extends CrudRepository<Invite, Long> {
 
     Invite findByForEmail(String forEmail);
 
     Invite findByCode(String code);
 
+    Optional<Invite> findByForEmailAndStatus(String code, InviteStatus status);
```

Comment:
Are the parameter name and method name matching? `findByForEmailAndStatus` with `String forEmail` or `findByCodeAndStatus` with `String code` ?

Reply:
DONE.
Review comment is applied. Please refer PR:
https://github.com/Civil-Service-Human-Resources/identity-service/pull/196/files